service/src/main/java/io/camunda/service/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -0,0 +1,183 @@
+    final var completed = filter.completed();
+    final var canceled = filter.canceled();
+
+    if (completed || canceled) {
+      return Arrays.asList("tasklist-task-8.5.0_alias");
+    } else {
+      return Arrays.asList("tasklist-task-8.5.0_");
+    }
+  }
+
+  private SearchQuery getVariablesQuery(final List<VariableValueFilter> variableFilters) {
+    if (variableFilters != null && !variableFilters.isEmpty()) {
+      final var transformer = getVariableValueFilterTransformer();
+      final var queries =
+          variableFilters.stream()
+              .map(transformer::apply)
+              .map((q) -> hasChildQuery("variable", q))
+              .collect(Collectors.toList());
+      return and(queries);
+    }
+    return null;
+  }
+
+  private SearchQuery getDateFilter(final DateValueFilter filter, final String field) {
+    if (filter != null) {
+      final var transformer = getDateValueFilterTransformer();
+      return transformer.apply(new DateFieldFilter(field, filter));
+    }
+    return null;
+  }
+
+  private SearchQuery getProcessInstanceKeysQuery(final List<Long> processInstanceKeys) {
+    return longTerms("processInstanceId", processInstanceKeys);
+  }
+
+  private SearchQuery getProcessDefinitionKeyQuery(final List<String> processDefinitionIds) {
+    return stringTerms("processDefinitionId", processDefinitionIds);
+  }
+
+  private SearchQuery getUserTaskKeysQuery(final List<Long> userTaskKeys) {
+    return longTerms("key", userTaskKeys);
+  }
+
+  private SearchQuery getUserTasksImplementationOnly() {
+    return term("implementation", "ZEEBE_USER_TASK");
+  }
+
+  private SearchQuery getCandidateUsersQuery(final List<String> candidateUsers) {
+    return stringTerms("candidateUsers", candidateUsers);
+  }
+
+  private SearchQuery getCandidateGroupsQuery(final List<String> candidateGroups) {
+    return stringTerms("candidateGroups", candidateGroups);
+  }
+
+  private SearchQuery getAssigneesQuery(final List<String> assignee) {
+    return stringTerms("assignee", assignee);
+  }
+
+  private SearchQuery getStateQuery(final List<String> state) {
+    return stringTerms("state", state);
+  }
+
+  private SearchQuery getTenantQuery(final List<String> tenant) {
+    return stringTerms("tenantId", tenant);
+  }
+
+  private SearchQuery getBpmnProcessIdQuery(final List<String> bpmnProcessId) {
+    return stringTerms("bpmnProcessId", bpmnProcessId);
+  }
+
+  // TO-DO: Possible values: [CREATED, COMPLETED, CANCELED, FAILED]
+  private SearchQuery getCreatedQuery(final boolean created) {
+    if (created) {
+      return term("state", "CREATED");
+    }
+
+    return null;
+  }
+
+  private SearchQuery getCompletedQuery(final boolean completed) {
+    if (completed) {
+      return term("state", "COMPLETED");
+    }
+
+    return null;
+  }
+
+  private SearchQuery getCanceledQuery(final boolean canceled) {
+    if (canceled) {
+      return term("state", "CANCELED");
+    }
+
+    return null;
+  }
```

Comment:
These methods are not used, and the todo is unclear to me. Can it be deleted for the time being?

Reply:
True, I will delete for now :)